src/components/core/TextArea/TextArea.tsx

```diff
@@ -30,6 +30,47 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
     },
     ref
   ) => {
+    // If the user is pasting a link, automatically place it in a Markdown link
+    const handleKeyDown = useCallback(
+      async (event) => {
+        if (event.key === 'v' && event.metaKey) {
+          // Wrap in a try/catch because navigator.clipboard.readText() is not supported in Firefox. If it fails, pasting behavior will be default
+          try {
+            // FIXME: Handle Typescript error from forwardRef
+            if (ref.current) {
```

Comment:
Need help on resolving this `forwardRef` typescript error that I have encountered before but still do not understand 😐

Reply:
cc @tbezman the typescript overlord

Reply:
I'll check this out Monday, unless urgent. let's wait to merge